File: WebSource/WebSource/Models/JSale.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebSource.Models
{
    public class JSale
    {
        public SProduct[] products { get; set; }
        public String cust_name { get; set; }
        public String cust_phone { get; set; }
        public double amount_paid { get; set; }
        public double total_amount { get; set; }
        public double discount { get; set; }

    }
}

File: androidApp/POS/app/src/main/java/com/apptriangle/pos/sales/response/JSale.java
package com.apptriangle.pos.sales.response;

import com.apptriangle.pos.model.Product;

import java.util.List;

/**
 * Created by zeeshan on 5/8/2018.
 */
public class JSale {
    public int invoiceId;
    public String str_invoiceId ;
    public int shopID ;
    public String agentID ;
    public String cust_name ;
    public String cust_phone ;
    public String dlr_details ;
    public String apiKey ;
    public List<Product> products ;
    public double amount_paid ;
    public double total_amount ;
}
